### PR TITLE
Upgrade to core13.4

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,16 +3,17 @@
 **This project is in Release Candidate stage.**
 
 ### Enhancements
-* None
+* Improve performance of interprocess mutexes on iOS which don’t need to support reader-writer locking. The primary beneficiary of this is beginning and ending read transactions, which is now almost as fast as pre-v13.0.0 (Core upgrade).
```
Comment (nielsenko) [inline CHANGELOG.md:6]: From previous experience I believe this should go under fixed. We are not enhancing the dart sdk. See discussion for upgrade to v13.3.0

Comment (desistefanova) [reply CHANGELOG.md:6]: Comparing to the [core changelog](https://github.com/realm/realm-core/blob/master/CHANGELOG.md) it is enhancement. Are you sure we have to move it under Fixes?

Comment (blagoev) [reply CHANGELOG.md:6]: The difference is very subtle but having this in Fixes is a tid better imo. Previously we decided to move it to fixes

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,16 +3,17 @@
...
 
 ### Fixed
 * None
 
 ### Compatibility
 * Realm Studio: 13.0.0 or later.
+* Fileformat: Generates files with format v23. Reads and automatically upgrade from fileformat v5 (Core upgrade).
```
Comment (blagoev) [inline CHANGELOG.md:13]: I think we should remove this line? We mentioned it on the Core 13 upgrade changelog,  but now we are just bumping a minor version of Core.

**Review by blagoev — APPROVED** [review]
+1 on @nielsenko's comment

**Review by nielsenko — APPROVED** [review]

**Pull request merged**